modules/vba_documents/app/sidekiq/vba_documents/upload_processor.rb

```diff
@@ -94,17 +96,21 @@ def original_file_metadata(tempfile)
       }
     end
 
-    def find_icn(file_number)
-      bgss = BGS::Services.new(external_uid: file_number, external_key: file_number)
+    def find_icn(parts)
+      consumer_file_number = read_original_metadata_file_number(parts)
 
-      # File Number is ssn, file number, or participant id.  Call BGS to get the
-      # veterans birthdate
+      return if consumer_file_number.blank?
```

Comment:
I'm a little confused why this variable is called `consumer_file_number` rather than simply `file_number`. It made me question whether we do something to modify the file number (causing a potential difference between a consumer-provided file number and the file number we send the EMMS API), but I confirmed that we do not.

Reply:
not sure why i did that....i guess i was trying to indicate it is the file_number as provided by the consumer..idk..changed it though....

Reply:
renamed it to file_number